v2/dtos/event_test.go

```diff
@@ -22,6 +22,9 @@ func TestFromEventModelToDTO(t *testing.T) {
 		DeviceName: TestDeviceName,
 		Created:    TestTimestamp,
 		Origin:     TestTimestamp,
+		Tags: map[string]string{
+			"GatewayID": "Intel123",
```

Comment:
The test data doesn't match the expected data
`"GatewayID": "Houston-0001"`

Reply:
Humm, thought I changed all of them to `Houston-0001`... :-(

Fixed and all tests passing in my branch. :-)